.github/workflows/ci.yml

```diff
@@ -67,39 +67,59 @@ jobs:
 
           # Build optee_os and optee_client for qemu_v8
           ./build_optee_libraries.sh $HOME
-
-          # Setup environment
           export OPTEE_DIR=$HOME
-          source environment
 
-          # Build OP-TEE Rust examples for Arm 64-bit both host and TA
+          # Build OP-TEE Rust examples for Arm 32-bit both host and TA
+          export ARCH_HOST=arm
+          export ARCH_TA=arm
+          source environment
           make -j`nproc`
 
-          # Build OP-TEE Rust examples for Arm 64-bit host and 32-bit TA
```

Comment:
I see you have dropped mixed build tests with host for 64-bit and TA for 32-bit or vice-versa. Any particular reason to do so?

Reply:
The host and TA are built separately for each example, seems building both aarch64 and both arm32 cover all cases. If `building host as aarch64 and TA as arm32` doesn't introduce new case I think it's okay to remove them

Reply:
The reason for me to add those tests was to be confident that our top level scripts or Makefile does proper required configuration to handle every case. I suppose it won't be much cumbersome for the CI.

Reply:
added